envs/buildroot/README.md

```diff
@@ -0,0 +1,15 @@
+# buildroot
+
+Buildroot, making embedded Linux easy.
+
+For more info checkout [buildroot over on GitLab](https://gitlab.com/buildroot.org/buildroot) or on their [website](https://buildroot.org).
+
+## Remarks
+
+- I was not able to get this working on 23.11, have yet to try it on 24.05
```

Comment:
So what version works, unstable only?

Reply:
Yes, since on 23.11 a week ago I tried it on unstable first (not on 24.05 yet) and it worked I opted to let this environment use the unstable branch via fetchTarball

Reply:
```suggestion
- works on unstable, but not on 23.11. 24.05 has not been tested.
```